raphtory-graphql/src/model/graph/graph.rs

```diff
@@ -368,7 +314,7 @@ impl GqlGraph {
     ////////////////////////
 
     async fn name(&self) -> String {
-        self.name.clone()
+        get_graph_name(&self.path)
```

Comment:
add a test for nested paths with disk graph for this

Reply:
We need to add many things for DiskGraphs - as upload etc not currently working - we can do as a followup once the cachegraph is in